health/egov-persister/plan-service-persister.yml

```diff
@@ -494,3 +494,42 @@ serviceMaps:
             - jsonPath: $.Plan.auditDetails.lastModifiedBy
 
             - jsonPath: $.Plan.auditDetails.lastModifiedTime
+
+    - version: 1.0.0
+      description: Persists the plan facility
+      fromTopic: save-plan-facility
+      isTransaction: true
+      isAuditEnabled: true
+      queryMaps:
+        - query: INSERT INTO plan_facility_linkage(id, tenant_id, plan_configuration_id, facility_id, residing_boundary, service_boundaries, additional_details, active ,created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,?);
```

Comment:
?,?); in consistent spacing
 , active ,

Reply:
Made correct spacing